.clabot

```diff
@@ -58,7 +58,8 @@
     "sudheerDev",
     "SVell",
     "JoaoCampos89",
-    "jchanolm"
+    "jchanolm",
+    "0xTxbi"
```

Comment:
@0xTxbi Please move this change to another PR!

Reply:
We can't do it for you because of course that would not mean you agree to it :)

Reply:
very true. done. moved it to another PR here: #13432